### PR TITLE
docs: update consensus-node page 

### LINKED ISSUE #1640: Ideas for full-consensus-node page
While working on a separate issue, I observed a few things on: https://docs.celestia.org/nodes/full-consensus-node. Ideas for improvement:

- Update title to: Full consensus node 
- "hardware minimum requirements" -> "minimum hardware requirements"
- "If you are not using pruning, you are running an archive node, and it is recommended to have 500 GB of SSD storage. Please ensure that your storage meets this requirement to ensure smooth syncing and operation of the node." I think this line needs to be updated because I ran a node that synced from genesis without pruning and needed 1300 GB.
- "Setting up a full consensus node" -> Set up a consensus full node
- We should be consistent about usage of "full consensus node" vs. "consensus full node"
- Subheadings don't need "-ing"
- I don't think we need to clone the networks repo anymore because the `download-genesis` command
- Explain what persistent peers do
- Explain (or link) to a sentry node
- "Connecting a consensus node to a bridge node" section should be marked optional
- "Accessing historical state" section should be marked optional.
- https://docs.celestia.org/nodes/full-consensus-node#syncing we should provide time estimates for each of these three options. Block sync will take several weeks. Block sync should also state something like: "from genesis up to the tip"
- "Refer to [the ports section of the celestia-node troubleshooting page](https://docs.celestia.org/nodes/celestia-node-troubleshooting#ports) for information on which ports are required to be open on your machine." Why do we need a link to celestia-node ports section here? Can we remove this tip?

### LINKED ISSUE #1650: docs: update storage requirements for full storage nodes / validators
# Problem
Docs requirements are too low for full state sync.

see https://snaps.qubelabs.io/celestia/
- mainnet is 1.6 TB
- mocha is 1.1 TB

# Solution
increase storage requirements to 2 TB

### LINKED ISSUE #1661: docs: don't use Cosmovisor
In a call, @evan-forbes suggested updating the docs for v2 to state that Cosmovisor isn't needed and consensus nodes can be upgraded before the upgrade height.

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

Resolves #1640

## Summary by CodeRabbit

- **New Features**
	- Enhanced hardware requirements for consensus nodes to improve performance capabilities.

- **Bug Fixes**
	- Simplified language in multiple documents to improve clarity without affecting functionality.

- **Documentation**
	- Updated terminology for various node types, changing "full consensus node" to "consensus node" for consistency.
	- Streamlined guidance in documentation related to consensus nodes, including clearer instructions and descriptions across various topics.

### CODE REVIEW

File: nodes/full-consensus-node.md [hunk 1]
```diff
@@ -17,25 +17,17 @@ consensus layer.
 
 ![full consensus node](/img/nodes/full-consensus-node.png)
 
-## Hardware requirements
+## Minimum hardware requirements
 
 The following hardware minimum requirements are recommended for running a
```
Comment (rootulp) [inline nodes/full-consensus-node.md:22]:
```suggestion
The following minimum hardware requirements are recommended for running a
```

**Comment by jcstein** [comment]
Moving update of graphics to #1650 for now

File: nodes/consensus-node.md [hunk 2]
```diff
@@ -199,15 +198,22 @@ pruning = "everything"
...
+## Sync types
+
+| Mode              | Block sync  | State sync      | Quick sync |
+|-------------------|-------------|-----------------|------------|
+| Time              | ~3 weeks    | TBD             | TBD        |
+| Trust assumptions | Most secure | Trusts snapshot | TBD        |
+
+### Option 1: Block sync
 
 By default, a consensus node will sync using block sync; that is request, validate
```
Comment (jcstein) [inline nodes/consensus-node.md:210]:
```suggestion
By default, a consensus node will sync using block sync; which will request, validate
```

File: nodes/consensus-node.md [hunk 3]
```diff
@@ -1,41 +1,32 @@
...
 <!-- markdownlint-disable MD033 -->
 <!-- markdownlint-disable MD013 -->
 <script setup>
 import constants from '/.vitepress/constants/constants.js'
 </script>
 
-This guide covers how to set up a full consensus node on Celestia.
-Full consensus nodes allow you to sync blockchain history in the Celestia
+This guide covers how to set up a consensus node on Celestia.
+Cconsensus nodes allow you to sync the entire blockchain history in the Celestia
```
Comment (rootulp) [inline nodes/consensus-node.md:15]:
```suggestion
Consensus nodes allow you to sync the entire blockchain history in the Celestia
```

File: nodes/consensus-node.md [hunk 4]
```diff
@@ -199,15 +198,22 @@ pruning = "everything"
 min-retain-blocks = 0 # this is the default setting
 ```
 
-## Syncing
+## Sync types
+
+| Mode              | Block sync  | State sync      | Quick sync |
+|-------------------|-------------|-----------------|------------|
+| Time              | ~3 weeks    | TBD             | TBD        |
+| Trust assumptions | Most secure | Trusts snapshot | TBD        |
```
Comment (rootulp) [inline nodes/consensus-node.md:206]: I'm collecting this data now. Should we hold off on merging this table with TBDs? I can add it back after I have the data.

```suggestion
```

File: nodes/consensus-node.md [hunk 5]
```diff
@@ -199,15 +198,22 @@ pruning = "everything"
...
 
-By default, a consensus node will sync using block sync; that is request, validate
+### Option 1: Block sync
+
+By default, a consensus node will sync using block sync; which will request, validate
 and execute every block up to the head of the blockchain. This is the most secure
 mechanism yet the slowest (taking up to days depending on the height of the blockchain).
```
Comment (rootulp) [inline nodes/consensus-node.md:212]:
```suggestion
mechanism yet the slowest (taking up to weeks depending on the height of the blockchain).
```

**Review by rootulp — APPROVED** [review]
Thanks a ton!

File: nodes/overview.md [hunk 6]
```diff
@@ -29,7 +29,25 @@ Data Availability:
...
+
+## Recommended Celestia node requirements
+
+## Data availability nodes
+
+| Node type         | Memory      | CPU         | Disk       | Bandwidth |
+|-------------------|-------------|-------------|------------|-----------|
+| Light node        | 500 MB RAM  | Single core | 100 GB SSD | 56 Kbps   |
+| Bridge node       | 16 GB RAM   | 6 cores     | 10 TB SSD  | 1 Gbps    |
+| Full Storage node | 16 GB RAM   | Quad-core   | 10 TB SSD  | 1 Gbps    |
```
Comment (jcstein) [inline nodes/overview.md:42]:
```suggestion
| Full storage node | 16 GB RAM   | Quad-core   | 10 TB SSD  | 1 Gbps    |
```

**Comment by jcstein** [comment]
you're welcome @rootulp !

**Pull request merged**